blackbird_python/blackbird/tests/test_utils.py

```diff
@@ -48,12 +48,31 @@ def test_no_args(self):
         assert nodes[0]['args'] == []
         assert nodes[0]['kwargs'] == {}
 
+    def test_no_dependence(self):
+        """Test case where operations do not depend on each other.
+        This should result in a graph with no edges."""
+        program = loads(dedent(
+            """\
+            name prog
+            version 0.0
+
+            Sgate(0.43) | 0
+            Dgate(0.543) | 1
+            """
+        ))
+
+        res = to_DiGraph(program)
+        assert len(res) == 2
+
+        nodes = res.nodes().data()
+        assert not list(res.edges())

```

Comment:
Since this is a test, I would make it more explicit: `assert len(res.edges()) == 0`

Reply:
```suggestion
        assert len(res.edges()) == 0
```